apps/advisor/pkg/app/checks/plugincheck/check.go

```diff
@@ -146,11 +156,19 @@ func (s *updateStep) Run(ctx context.Context, _ *advisor.CheckSpec, i any) (*adv
 
 	// Skip if it's a core plugin
 	if p.IsCorePlugin() {
+		s.log.Debug("Skipping core plugin", "plugin", p.ID)
 		return nil, nil
 	}
 
 	// Skip if it's managed or pinned
 	if s.isManaged(ctx, p.ID) || s.PluginPreinstall.IsPinned(p.ID) {
+		s.log.Debug("Skipping managed or pinned plugin", "plugin", p.ID)
+		return nil, nil
+	}
+
+	// Skip if it's provisioned
+	if s.isProvisioned(ctx, p.ID) {
```

Comment:
This will call out to GCOM every time the check is evaluated and for each plugin, right? Should we maybe cache the response it for a period of time at least so that we limit outgoing requests?